script.js

```diff
@@ -14,3 +14,173 @@ Items.forEach((n) => {
     dropdownMenu.classList.toggle('hide');
   });
 });
+
+// popup!!!
+
+const body = document.querySelector("body");
+const section = document.createElement("section");
+body.appendChild(section);
+section.innerHTML = `
+  <div class="popup pophide">
+  <div class="phead">
+  <img src="images/pop.svg" alt="" class="pop-img">
+  <span class="xsign"><img src="images/xicon.svg" alt=""></span>
+  </div>
+  <h2 class="phead">Keeping track of hundreds of
+    components</h2>
+    <ul class="plist">
+      <li><a href="#" class="panc card-a4">Ruby on rails</a></li>
+      <li><a href="#" class="panc card-a2">CSS</a></li>
+      <li><a href="#" class="panc card-a3">Javscript</a></li>
+    </ul>
+  <p class="ptext">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+  <ul class="pbuttons">
+  <button class="see" type="button" name="button">
+  <span>See Live</span>
+  <span><img src="images/upload.svg" alt=""></span>
+  </button>
+  <button class="see" type="button" name="button">
+  <span>See Source</span>
+  <span><img src="images/vector.svg" alt=""></span>
+  </button>
+  </ul>
+  </div>
```

Comment:
- [ ] Kindly match your design with the Figma design. The popup should follow the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colours, images, text, margins). Also, the popup is responsive, it fits mobile and desktop sizes properly.

Your design [Desktop view]
:-------------------------:
![popup_desktop_version2](https://user-images.githubusercontent.com/70483835/208957653-941dd4f8-4997-446e-afc0-b7d2f526b594.png)

Figma Design [Desktop view]
:-------------------------:
![popup_desktop_version1](https://user-images.githubusercontent.com/70483835/208959226-3e8a7a70-acc6-4445-8ee8-dd680b1dbb85.png)

Your design [Mobile view]
:-------------------------:
![mobile_version2](https://user-images.githubusercontent.com/70483835/208957415-7cf45dfa-afd0-43f2-aea0-24c5a9806a82.png)

Figma Design [Mobile view]
:-------------------------:
![mobile_version1](https://user-images.githubusercontent.com/70483835/208957388-8c9f8eeb-ee07-4487-ad9d-382851ecfd1a.png)